src/main.ts

```diff
@@ -7,7 +7,14 @@ async function bootstrap() {
   const app = await NestFactory.create<NestExpressApplication>(AppModule);
   app.enableCors();
   app.set("trust proxy", 1);
-  app.useGlobalPipes(new ValidationPipe({ whitelist: true }));
+  app.useGlobalPipes(
+    new ValidationPipe({
+      whitelist: true,
+      transformOptions: {
+        enableImplicitConversion: true,
+      },
```

Comment:
I am a bit curious about this part. Could you please expain it? I didn't find it in the Nestjs documentation but just some not implicit examples on the Internet. I will keep doing research on the Internet but I still expect your answer.

Reply:
This one for pagnation